operators/pkg/controller/kibana/kibana_controller.go

```diff
@@ -131,9 +138,16 @@ func (r *ReconcileKibana) Reconcile(request reconcile.Request) (reconcile.Result
 		log.Info("End reconcile iteration", "iteration", currentIteration, "took", time.Since(iterationStartTime))
 	}()
 
+	// reconcile and set operator uuid
+	uuid, err := about.ReconcileOperatorUUIDConfigMap(r.operatorClient, r.scheme, r.operatorInfo.Namespace)
```

Comment:
Does this need to happen on every reconcile loop iteration? Could we not just run it once on startup and then hold the UID in memory?

Reply:
I have no argument against. It will be simpler and a getOrCreate must be enough instead of the reconcilation loop (also, no need for the client cache).